File: 1021-remove-outermost-parentheses/1021-remove-outermost-parentheses.py
class Solution:
    def removeOuterParentheses(self, s: str) -> str:
        temp=""
        c=0

        for ch in s:
            if ch=='(':
                c+=1
                if c>1:
                    temp+=ch
            else:
                if ch==')':
                    c-=1
                    if c!=0:
                        temp+=ch

        return temp

File: 1021-remove-outermost-parentheses/1021-remove-outermost-parentheses.js
/**
 * @param {string} s
 * @return {string}
 */
var removeOuterParentheses = function(s) {
        let count=0;
        let t="";
        for (let num of s){
            if (num==='('){
                count++;
                if (count>1){
                    t+='(';
                }
            }else{
                count--;
                if (count>0){
                    t+=num;
                }
            }
        }
        return t; 
       
       
};